### PR TITLE
Set arg x for xklb db path

### PR DESCRIPTION
**Pull request by deldesir**
Fix: #248

### CODE REVIEW

File: cps/cli.py [hunk 1]
```diff
@@ -76,18 +77,23 @@ def arg_parser(self):
                                                             'in advance and exits Calibre-Web')
         parser.add_argument('-r', action='store_true', help='Enable public database reconnect '
                                                             'route under /reconnect')
+        parser.add_argument('-x', metavar='path', help='path and name to xklb db')
```
Comment (avni) [inline cps/cli.py:80]:
```suggestion
        # this set of changes adds a new config parameter for the experimental xklb.db file to be used by lb wrapper and makes the path available in the Calibre-Web template: https://github.com/iiab/iiab/blob/master/roles/calibre-web/templates/calibre-web.service.j2
        parser.add_argument('-x', metavar='path', help='path and name to xklb db')
```

**Review by avni — CHANGES_REQUESTED** [review]
Added a comment to explain the addition of the config param.

**Comment by holta** [comment]
Is this PR truly needed, and if so why?

- "Parameter Pollution" (i.e. soft-coding of variables / flags / options prior to a proven need) is so often tempting but... far too often creates maintenance burdens down the road... _rather than helping a broad demographic?_

- Conversely, if the need for another parameter is in fact very real, for a specific QA/Community demographic or any other, then let's please all help articulate that in coming days.  (Either Way!)  And if the need is confirmed truly real — let's explain why parametrization is genuinely necessary here (e.g. justifying the inevitable ~ additional maintenance/testing/security burdens) ~ _ensuring we avoid Complexity Bloat..._

> [Premature Generalization, like...] Premature Optimization is the Root of All Evil
> —Donald Knuth (1974)

**Comment by deldesir** [comment]
Parametrization is not necessary nor needed. KISS principle.

**Pull request closed without merging**